docs/e2e-ci.md

```diff
@@ -27,7 +27,7 @@ A Bats test file is a Bash script with special syntax for defining test cases. U
 }
 ```
 
-If every command in the test case exits with a 0 status code (success), the test passes. This workflow uses bats framework to execute end to end tests, these tests are defined in files with the termination *.bats under the [e2e directory](./e2e)
+If every command in the test case exits with a 0 status code (success), the test passes. This workflow uses bats framework to execute end to end tests, these tests are defined in files with the termination *.bats under the [e2e directory](../e2e)
```

Comment:
I'd remove the ../e2e reference as it really can only add confusion when the user may be reading from a dir other than docs. If you want to point to it I'd use something like [e2e directory](benchmark-operator/2e2)